web/src/app/approved-researcher/components/ApprovedResearcherForm.tsx

```diff
@@ -0,0 +1,86 @@
+"use client";
+
+import { useState } from "react";
+
+import "./ApprovedResearcherForm.css";
+
+export default function ApprovedResearcherForm() {
+  const [submitted, setSubmitted] = useState(false);
+  const [formData, setFormData] = useState({
+    name: "",
+    email: "",
+    userID: "",
+    department: "",
```

Comment:
don't think we need any of this 😄

Reply:
do we need their EntraID? and we said we wanted a display name they can control, but not sure if that's this form

Reply:
the backend already has their EntraID. agree the display name should probably be before this form – probably be prompted on first access?

Reply:
so we probably only need one from the email and entra ID and this can be hidden from the user but can still be part of the form to link the doc to them. Unless the way we store the docs means they're already linked? (eg `docs/<entraID>/approved-researcher`)

Reply:
Ah ok, I took the fields from [this thing called 'Approved Researcher form'](https://liveuclac.sharepoint.com/sites/ARCTrustedResearch-ProdSharePoint/_layouts/15/listforms.aspx?cid=YTc4YTdmMTctMjJlOC00OTAyLTg1NDktMmRmOWVkMmQ3ZGI0&nav=NjIyZmQzNGYtOGEyMC00YzdlLTgzYmMtYTkxM2MwNTRjZmRj) 😅

Reply:
As this PR is not really about completing the approved researcher process, I'd like to focus on that in a follow-up PR to keep this strictly about setting up a basic layout. Unless anyone objects?